Unreal/SystemLocalized/det/DmRetrospective.det

```diff
@@ -11,7 +11,7 @@ LevelEnterText="Karte für den Unreal 227 Patch!"
 
 [MegaHealth3]
 ; EN: PickupMessage="You picked up the Mega Health :)"
-PickupMessage="Du hast die Mega Health aufgenommen :)"
+PickupMessage="Mega-Gesundheitspack erhalten :)"
 
 [LevelSummary]
 ; EN: Title="Retrospective"
```

Comment:
Suggestion: "Retrospektive" as name? It sounds as appropriate as possible and is an actual term.